src/main/java/TaskType.java

```diff
@@ -0,0 +1,4 @@
+public enum TaskType
+{
+    TO_DO, DEAD_LINE, EVENT
```

Comment:
Perhaps you could check if it is spelled deadline or dead line. The underscore between "DEAD" and "LINE" affects code readability as It might be interpreted as "dead line" which might be interpreted differently from "deadline". The same applies to "To_Do".

```suggestion
    TODO, DEADLINE, EVENT
```